com.unity.cinemachine/Documentation~/CinemachineUpgradeFrom2.md

```diff
@@ -6,22 +6,22 @@ While it is possible to upgrade an existing project from CM 2.X, you should thin
 
 ## Upgrading your Project Step by Step
 Here are the steps to take when upgrading an existing project from CM 2.X:
-1. Back up your project.  Don't skip this step.
```

Comment:
You really don't have to bother eliminating the double spaces after the periods.  The MD formatter ignores this, results are the same either way.